enterprise/functions/src/test/java/io/crate/window/OffsetValueFunctionsTest.java

```diff
@@ -101,7 +101,7 @@ public void testLagNegativeOffsetAndEmptyOver() throws Throwable {
     public void testLagWithDefaultValueAndEmptyOver() throws Throwable {
         assertEvaluate(
             "lag(x, 1, -1) over()",
-            contains(new Object[]{-1, 1, 2, 3}),
```

Comment:
not sure about this specific case, though.

Reply:
The new outcome is fine, the signature binder don't support implicit literal downcasts which the old FuncParams did. This is known and should be re-added later on on a different level as this is mostly lucene related.